Bottom: 5034802b08a8450a8bb3975d63e6de0b427bf598
Top:    2c33c9a1028295f905e3773c10d615145cf5cc0e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 16:34:25 +0100

Refresh of hid-gerber-don-t-use-the-core-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 4518b21..d521474 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -555,7 +555,7 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
     DrawPaste (SOLDER_LAYER, drawn_area);
 
   if (set_layer ("fab", SL (FAB, 0), 0))
-    PrintFab ();
+    PrintFab (Output.fgGC);
 
   gui->destroy_gc (Output.fgGC);
   gui->destroy_gc (Output.bgGC);
